src/js/components/LoginMessage.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Header from './Header.jsx';
+
+export default class LoginMessage extends React.Component {
+  /*
```

Comment:
Write this as a functional component since all it does is render the component. Refer to the comment in `src/js/components/Footer.jsx`